MAINTAINERS.yml

```diff
@@ -2768,11 +2759,12 @@ NIOS-2 arch:
     - arch/nios2/
     - dts/nios2/intel/
     - boards/common/nios2.board.cmake
-    - boards/nios2/
-    - soc/nios2/
+    - soc/altera/zephyr_nios2f/
+    - soc/altera/qemu_nios2/
```

Comment:
soc/altera ?

Reply:
adjusted to `soc/altera/*nios2*/` in case other non-nios2 altera socs come.